modules/cfe_testcase/src/cfe_test.c

```diff
@@ -59,6 +59,7 @@ void CFE_TestMain(void)
     FSHeaderTestSetup();
     SBPipeMangSetup();
     TimeCurrentTestSetup();
+    TimeConversionTestSetup();
```

Comment:
Minor thing but would put conversion before current, trying to keep the run order alphabetical moving forward just to have some order.

Reply:
I've had this before, but I thought I should test if it works with the CFE_TIME_GetTAI(), for example, before testing if the times can be converted correctly.